infra/prod/_modules/app_services/app_service_openid_rp_example.tf

```diff
@@ -0,0 +1,67 @@
+module "appservice_openid_rp_example" {
+  source = "github.com/pagopa/terraform-azurerm-v3//app_service?ref=v7.76.0"
+
+  plan_type = "internal"
+  plan_name = "${var.product}-openid-rp-example-plan"
+  sku_name  = "S1"
+
+  name                = "${var.product}-openid-rp-example-app"
```

Comment:
I would change the name of this "service" (and its references) to `rp-example-app` (without `openid-`) to avoid problems with the length of resource names

Reply:
@lucacavallaro please take a look here [b57e1bb](https://github.com/pagopa/io-fims/pull/22/commits/b57e1bb6d7d371192de73d31e884bc8b9c38d93f)